examples/get-started/pure-js/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@deck.gl/core": "^5.2.0",
+    "@deck.gl/layers": "^5.3.0-alpha",
```

Comment:
`alpha.2`?

Reply:
This is valid semver syntax. We'll update it to `^5.3.0` when it's published.

Reply:
OK. Does it actually do what we want (resolve to latest alpha), or just another way to do same old things? If better, should we use it across the board then? There is at least one more js example.